taskcluster/ci/browsertime/kind.yml

```diff
@@ -36,7 +36,7 @@ job-defaults:
                     subject: '[{product_name}] Raptor-Browsertime job "{task_name}" failed'
                     to-addresses: [perftest-alerts@mozilla.com]
             default: {}
-    run-on-tasks-for: []
+    run-on-tasks-for: [github-pull-request]
```

Comment:
Nit: I guess we don't want to run these tests on all PRs. I'm sure you know, I'm just making it explicit: Let's remove change before landing 🙂